frontend/src/components/pages/profile/Profile.tsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
-import Layout from '../../shared/layout/Layout';
 import User from '../../mocks/user-mock.json';
-import { UserCard } from '../../shared/userCard/UserCard';
+import Layout from '../../shared/layout/Layout';
+import UserCard from '../../shared/userCard/UserCard';
+import About from './components/about/About';
+import AddPostForm from './components/addPostForm/AddPostForm';
+import FriendsList from './components/friendsList/FriendsList';
+import Posts from './components/posts/Posts';
+import './profile.scss';
 
 const Profile: React.FC = () => {
   return (
     <Layout pageTitle="Profile">
-      <UserCard user={User} />
+      <div className="profile">
+        <div className="leftside">
+          <UserCard mainInfo={User} />
```

Comment:
`user` is variable/data, and not class, so it should be written with lowerCamelCase